openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1449,6 +1451,16 @@ Access Tokens represent End-User authorization and consent to issue certain Cred
 
 To sender-constrain Access Tokens, see the recommendations in (#securitybcp). If Bearer Access Tokens are stored by the Wallet, they MUST be stored in a secure manner, for example, encrypted using a key stored in a protected key store.
 
+## Application-Layer Encryption {#encryption-security-considersations}
+
+Depending on the architecture of the Wallet and the Issuer, additional encryption of requests and responses may provide additional confidentiality for data beyond TLS, at the cost of increased complexity. An example is when a Wallet is composed of different components with differing levels of trust (such as a Wallet server backend and client application). The same may apply to complex Issuer systems.
```

Comment:
```suggestion
Depending on the architecture of the Wallet and the Issuer, adding encryption to requests and responses, beyond transport-level security (TLS), can offer enhanced data confidentiality. This can come at the cost of added complexity. This approach can be particularly relevant when the Wallet consists of multiple components with varying trust levels, such as a backend server and a client application. Similar considerations may apply to more complex Issuer architectures.
```
trying to make it slightly smoother